apps/src/storage/dataBrowser/DataTableView.jsx

```diff
@@ -122,7 +123,7 @@ class DataTableView extends React.Component {
               onClick={() => onViewChange(DataView.OVERVIEW)}
             >
               <FontAwesome icon="arrow-circle-left" />
-              &nbsp;Back to data
+              &nbsp;{msg.backToData()}
```

Comment:
Could you do quick manual check that this looks good in RTL languages? I'm worried the spacing might be off. 

Also, does the 'arrow-circle-left' look correct as well?

Reply:
I believe I have resolved this.